code/game/machinery/doors/windowdoor.dm

```diff
@@ -432,7 +432,7 @@
 
 /obj/machinery/door/window/brigdoor/security/holding
 	name = "holding cell door"
-	req_one_access = list(ACCESS_BRIG_ENTRANCE) //lawyer also gets brig_entrance
+	req_one_access = list(ACCESS_SECURITY)
```

Comment:
```suggestion
	req_one_access = list(ACCESS_BRIG_ENTRANCE)
```
Why remove lawyer access to this?

Reply:
because it's a holding cell used for mass holding people. Why would a lawyer who doesn't have a mindshield be able to open the door for them to escape? They aren't security.